dashboards/google-backupdr/backupdr-monitoring.json

```diff
@@ -0,0 +1,162 @@
+{
+  "displayName": "Backup and DR Monitoring",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "Backup and DR - Storage Pool Utilization",
```

Comment:
Since "Backup and DR" is already in the dashboard name, I think we can remove it from each of the individual chart titles.

Reply:
Updated the Title for the each charts.